agent/containers/images/gen-tags-from-rpm

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+url_prefix="${1}"
+dist="${2}"
+arch="${3}"
+
+if [[ "${dist}" == "centos-8" ]]; then
+    dist="epel-8"
+elif [[ "${dist}" == "centos-7" ]]; then
+    dist="epel-7"
+fi
+
+version_string="$(yum list -q --repofrompath pbench,"${url_prefix}/pbench/${dist}-${arch}" pbench-agent.noarch 2>/dev/null | awk 'FNR==2{print $2}')"
+if [[ -z "${version_string}" ]]; then
+    printf -- "Failed to fetch pbench-agent.noarch version string from '%s/pbench/%s-%s'\n" "${url_prefix}" "${dist}" "${arch}" >&2
+    exit 1
+fi
+
+# 0.69.3-1
```

Comment:
are this changes specific to this agent version?

Reply:
No, that is just an example version number to help understand how the parsing works.